village/src/components/SmurfForm.js

```diff
@@ -12,7 +13,15 @@ class SmurfForm extends Component {
 
   addSmurf = event => {
     event.preventDefault();
```

Comment:
So, I wanted to find out why your post wasn't updating. While I couldn't find an exact answer removing
`event.preventDefault()` has it refresh for you... (I'm starting to think that the _auto-update_ feature is actually the page refreshing after the form is submitted ).